web/api/v1/api.go

```diff
@@ -91,6 +92,19 @@ type alertmanagerRetriever interface {
 	DroppedAlertmanagers() []*url.URL
 }
 
+type alertRetreiver interface {
+	AlertingRules() []*rules.AlertingRule
+}
+
+type rulesRetreiver interface {
+	RuleGroups() []*rules.Group
```

Comment:
This seems a bit unsymmetrical, both alerts and rules live inside rule groups.

Reply:
@brian-brazil : i did it this way in order to separate out the functions for making the unit test setup cleaner. I can put both functions as part of one interface signature rather than using interface composition. Let me know if you which way you prefer .